tasks/Dockerfile

```diff
@@ -4,7 +4,6 @@ RUN apt-get update &&\
     apt-get reinstall -y gdal-bin=3.2.2+dfsg-2+deb11u2 \
     libgdal-dev=3.2.2+dfsg-2+deb11u2 && \
     apt install -y wget unzip python3-rtree python3-all-dev tesseract-ocr ghostscript
-
```

Comment:
Had removed tesseract/ghostcript OCR dependencies but added back once "OCR Fallback" was added.

Dojo documents processing now supports two operating modes:
a) Use an external OCR engine
b) Use the original, aka fallback, OCR engine, which is somewhat inferior but will get the job done without needing a GPU to be available.